services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/cache/message/ICachePublisher.java

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2019 The Alcor Authors.
```

Comment:
See quite a few of source codes are from /lib, for example, ICachePublisher RedisListener etc. They should stay in /lib. 
We just use them (as you've imported the AlcorCommonLib), instead of duplicating the source codes here.

Reply:
fixed in 136f4b5